charts/consul/values.yaml

```diff
@@ -1742,7 +1742,7 @@ syncCatalog:
     # If mirroring is enabled, avoid creating any Consul resources in the following 
     # Kubernetes namespaces, as Consul currently reserves these namespaces for 
     # system use: "system", "universal", "operator", "root". 
-    mirroringK8S: false
+    mirroringK8S: true
```

Comment:
we need to set connectInject one too

Reply:
I am going to leave mirroring K8S for another PR because it opened a bigger Pandora's box of issues than I originally anticipated.